### PR TITLE
Fixed handling of urlPath containing an empty string

### PR DESCRIPTION
**Pull request by jonasraoni**

### CODE REVIEW

**Comment by jonasraoni** [comment]
@asmecher Just a minor fix (`urlPath` could be an empty string before some changes I did on the main branch).

File: GoogleScholarPlugin.php [hunk 1]
```diff
@@ -101,10 +106,9 @@ public function submissionView($hookName, $args)
             $templateMgr->addHeader('googleScholarPublisher', '<meta name="citation_publisher" content="' . htmlspecialchars($context->getName($context->getPrimaryLocale())) . '"/>');
         }
 
-
         $publication = $submission->getCurrentPublication();
         $publicationLocale = $publication->getData('locale');
-        $submissionBestId = $publication->getData('urlPath') ?? $submission->getId();
+        $submissionBestId = $submission->getBestId();
```
Comment (bozana) [inline GoogleScholarPlugin.php:111]: Isn't this method deprecated?

Comment (jonasraoni) [reply GoogleScholarPlugin.php:111]: It is, and I did it on purpose 😅
Extracting the internals of that method doesn't look better (requires more knowledge/attention from the reader/writer), so it doesn't make sense for me 🤔

---

In fact, I'd like to propose an alternative for the unknown/untyped stuff that was introduced recently (e.g. `getData('anything')`), I think it's ok to keep it as an "extension", but it's too risky/easier to introduce typos, but that's a discussion for a later time.

**Comment by asmecher** [comment]
Thanks, @jonasraoni -- and noted, @bozana!

**Pull request merged**